Libs/Gtk3/Classes/FileChooserButton.cs

```diff
@@ -110,6 +105,6 @@ public FileChooserDialog Dialog
             ConstructParameter.With(DialogProperty, dialog)) {}
 
         public static FileChooserButton New(string title, FileChooserAction action)
-            => new FileChooserButton(Native.@new(title, action));
+            => new FileChooserButton(Native.@new(title, action), false);
```

Comment:
I believe this should be `true`?

[`gtk_file_chooser_button_new`](https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/gtk/gtkfilechooserbutton.c#L3016) doesn't have introspection attributes (which is probably a documentation issue), but as it's simply a wrapper over `g_object_new`, it should return an owned reference (e.g. `transfer full`).

Reply:
It is not documented in code, which looks like resulting in some default values in the gir file which states `transfer-ownership="none"` which is correct as it is derived from `GInitiallyUnowned `. In this case it looks like it is just a wrapper so you are probably right. I asked this in the #introspection chat. Let's see what is happening.

Reply:
The answer that the documentation is correct. g_object_new is not always transfering the full ownership. So my assumption was false but this concrete piece of code is correct.

Reply:
I think your solution to `ref_sink` if floating should be enough for now. We can revisit this later if it becomes relevant, but I don't think we'll need much more than that.